config/envs/development.py

```diff
@@ -22,3 +22,9 @@ class DevelopmentConfig(DefaultConfig):
 
     # Database
     SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+    DEV_SQLALCHEMY_DATABASE_URI = (
+        DefaultConfig.SQLALCHEMY_DATABASE_URI + "_DEV"
```

Comment:
Why are we changing this value? Don't think this is is consistent with what we do for other apps?

Reply:
I changed it so that using an invoke task would create a specially named db, although on reflection that can be set in the invoke task so that config isnt changed.